### PR TITLE
Adjustment code report

### PR DESCRIPTION
**Pull request by teryhill**
This report gives a total for the selected adjustment code over a given time period. #443

### CODE REVIEW

**Comment by aethelwulffe** [comment]
I'll get reviewing again tomorrow.  Flu got me down hard.

File: sites/default/menu_data.json [hunk 1]
```diff
@@ -988,41 +994,41 @@
         "global_req" : "addr_label_type"
     } ],
     "requirement" : 0
-}, {
+}, { 
     "label" : "QA Measures",
     "menu_id" : "mips",
-    "children" : [ {
+    "children" : [ {
```
Comment (aethelwulffe) [inline sites/default/menu_data.json]: How did the extra white pace come in?  Was that from your editor, or was it generated by the menu editor?  If it was from the menu editor, we should probably leave that as a standard.

File: interface/reports/adjustments.php [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+<body class="body_top">
+
+<div id="overDiv" style="position:absolute; visibility:hidden; z-index:1000;"></div>
+
+<span class='title'><?php echo xlt('Report'); ?> - <?php echo xlt('Adjustments'); ?></span>
+
+<div id="report_parameters_daterange">
+    <?php date("d F Y", strtotime(oeFormatDateForPrintReport($_POST['form_from_date'])))
+    . " &nbsp; to &nbsp; ". date("d F Y", strtotime(oeFormatDateForPrintReport($_POST['form_to_date']))); ?>
```
Comment (aethelwulffe) [inline interface/reports/adjustments.php]: Does this date format function allow for localization?  If not, then Y m d is preferred.  If using Portuguese as the language doesn't result in "Januario" then we are just making trouble.

Comment (teryhill) [reply interface/reports/adjustments.php]: It is the same date function in all the other reports so I hope so 8^)

**Comment by teryhill** [comment]
Pictures of what it looks like.

image

image

**Comment by aethelwulffe** [comment]
OK,  Works.

**Pull request merged**